Solution for polygon-area-calculator exercise

fesiddi opened the pull request.

storrisi reviewed (changes requested):
I just see the class implementation.

Maybe you can add some tests so your code could be used for real.

What do you think?

fesiddi commented:
Hi Simone, the readme file says to implement the Square and Rectangle classes in the shape_calculator.py file. All the test code is in the main.py and test_module.py files. I've added for review purpouses some test code in shape_calculator.py file and modified the solution file location inside 'solution/federicosiddi/' to keep the original shape_calculator.py file untouched. Let me know if i've done right!

storrisi commented:
There is a file outside your solutions directory, btw I don't see any main.py and test_module.py within this solution, maybe you missed to add them?

fesiddi commented:
Ok, i've left them outside of solutions folder on purpose but now i've fixed it.

storrisi commented on projects/m5/002-polygon-area-calculator/shape_calculator.py:
```diff
@@ -3,4 +3,4 @@ class Rectangle:
 
 
-class Square:
+class Square:
```
This file shouldn't be here
